lib/db/influx_v1.go

```diff
@@ -0,0 +1,81 @@
+package db
+
+import (
+	"fmt"
+	"net"
+)
+
+type InfluxDBV1Handler struct {
+	conn net.UDPConn
+	addr string
+}
+
+// Initialize sets up the InfluxDB UDP connection
+func (h *InfluxDBV1Handler) Initialize() error {
+	h.addr = "localhost:8089" // TODO: Make this IP and port configurable
+
+	addr, err := net.ResolveUDPAddr("udp", h.addr)
+	if err != nil {
+		fmt.Println("Error creating InfluxDB UDP client:", err)
+		return err
+	}
+
+	conn, err := net.DialUDP("udp", nil, addr)
+	if err != nil {
+		fmt.Println("Error creating InfluxDB UDP client:", err)
+		return err
+	}
+
+	h.conn = *conn
+	return nil
+}
+
+// CreateQuery Generates InfluxDB query for measurement group
+func (h *InfluxDBV1Handler) CreateQuery(measurements MeasurementGroup) string {
+	query := measurements.DatabaseName + " "
+
+	for _, measurement := range measurements.Measurements {
+		query += fmt.Sprintf("%s=%s,", measurement.Name, measurement.Value)
+	}
+
+	// Don't check if string is empty. We expect the Name and the measurements to be non-empty.
+	query = query[:len(query)-1]
+
+	// Add timestamp if it exists
```

Comment:
Why is this commented out?

Reply:
Was trying to make a decision on something